CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+<<<<<<< HEAD
+## [0.1.6] - 2023-10-03 20:00:00
+
+### Added
+
+- Adds Montana tax logic.
```

Comment:
@finpaul 
- Get rid of git merge conflict metadata
- Update version to 0.2.6, and update the date and time stamp
- Add the version compare line at the end of this file